specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -668,6 +668,50 @@
         }
       }
     },
+    "EncryptionDetails": {
+      "description": "Details of the encryption associated with the workspace",
+      "type": "object",
+      "properties": {
+        "doubleEncryptionEnabled": {
+          "description": "Double Encryption enabled",
+          "type": "boolean",
+          "readOnly": true
```

Comment:
if this is readonly is this just mimicking the cmk.status boolean?  When would this be false if cmk.status is true?

Reply:
No, because we're the evolution of SQL DW, we foresee having to support cases where doubleEncryption is enabled but CMK isn't (since this is already how they support it)

Reply:
It is marked as readonly.  How does a user enable it?

Reply:
They can't use it right now. The PMs in charge of this security feature haven't decided how this would work or if it would ever happen, but we want the user to know that the workspace is double encrypted or not